plugin/kubernetescrd/README.md

```diff
@@ -0,0 +1,161 @@
+# kubernetescrd
```

Comment:
Small note from reviewing this README:  maybe a few words or a section about security?

A malicious user could create these resources in order to steal traffic (as they could if they had write access to the `coredns` configmap in `kube-system`

May be worth recommending that users limit the namespace to `kube-system`, perhaps show that example first with some language recommending that?  Also worth noting that when touching on the RBAC topic, since RBAC on writes to the resource could be another way of locking down the security.

Reply:
Good point. I've added a small section to readme wrt to this.